spec/fabricators/upload_fabricator.rb

```diff
@@ -31,10 +31,11 @@
 
 Fabricator(:image_upload, from: :upload) do
   transient color: "white"
+  transient color_depth: 16
```

Comment:
Should the default here maybe be `8` since `16` is the outlier?

Reply:
oh yikes ... I thought I rolled that back .. fixing.

Reply:
Thanks! :+1: